satnogsclient/scheduler/tasks.py

```diff
@@ -30,9 +30,19 @@ def spawn_observation(*args, **kwargs):
 
 def get_jobs():
     """Query SatNOGS Network API to GET jobs."""
+
+    print 'Try getting Jobs from Network ..'
+
     url = urljoin(settings.NETWORK_API_URL, 'jobs')
     params = {'ground_station': settings.GROUND_STATION_ID}
-    response = requests.get(url, params=params)
+    try:
+        response = requests.get(url, params=params, verify=settings.CA_CERT)
```

Comment:
We need to somehow handle the case  when ``CA_CERT`` is not defined in settings.
* Add a default value in settings (``None``)
* Check if ``settings.CA_CERT`` is defined